### ISSUE TITLE
Don't use strings to compare IPO status

### ISSUE THREAD

**Issue by frankbryden** [frankbryden/ipo-calendar#2]
The status of an IPO is stored as a string, leading to multiple string comparisons later on. Ideally, this should be replaced by some sort of enum.

**Comment by maximilianvb** [comment]
Yes, this is a major problem that was created by me, I will fix it ASAP.
